README.md

```diff
@@ -206,9 +206,43 @@ pop up parachain -f ./tests/zombienet.toml -p https://github.com/r0gue-io/pop-no
 > :information_source: Pop CLI will automatically source the necessary polkadot binaries. Currently, these will be built
 > if on a non-linux system.
 
+
+## Testing Pop CLI 
+
+To test the tool locally.
+
+Run the unit tests:
+
+```sh
+cargo test
+```
+
+Run only contracts unit tests:
```

Comment:
nit: Some more information here can be useful. For eg: "Runs the unit tests within the ink smart contract"